pkg/drive/event.go

```diff
@@ -345,10 +350,92 @@ func (handler *driveEventHandler) handleUpdate(ctx context.Context, drive *types
 	return nil
 }
 
+func (handler *driveEventHandler) checkDrive(ctx context.Context, drive *types.Drive) error {
+	switch drive.Status.Status {
+	case directpvtypes.DriveStatusReady:
+		if err := handler.exists(types.GetVolumeRootDir(drive.Status.FSUUID)); err != nil {
+			if errors.Unwrap(err) == syscall.EIO {
+				if uerr := SetIOError(ctx, drive.GetDriveID()); uerr != nil {
+					klog.ErrorS(uerr, "unable to set I/O error", "drive", drive.GetDriveID())
+				}
+			} else {
+				klog.ErrorS(err, "unable to check volume root directory existent", "drive", drive.GetDriveID())
+			}
+
+			return nil
+		}
+
+		if _, err := handler.getDeviceByFSUUID(drive.Status.FSUUID); err != nil {
+			klog.ErrorS(
+				err,
+				"unable to find device by FSUUID; "+
+					"either device is removed or run command "+
+					"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
+					"on the host to reload",
+				"FSUUID", drive.Status.FSUUID)
+			client.Eventf(
+				drive, client.EventTypeWarning, client.EventReasonDeviceNotFoundError,
+				"unable to find device by FSUUID %v; "+
+					"either device is removed or run command "+
+					"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
```

Comment:
actually, `udevadm trigger` will not fix the symlink problems sometime. we may require a reboot in such cases. do we need to communicate this information as well?

Reply:
No. It is not a general case. `udevadm trigger` is general guideline whereas node reboot is specific case.